### DISCUSSION TITLE
period dates

### DISCUSSION THREAD

**Discussion by j0w13-0x9** [firefly-iii/firefly-iii#6688] [Firefly III]
Is it possible to change the period dates. For example if one gets their salary on the 26th of every month. I have noticed that December is a period, therefore it throws all the stats out, like budgets and the balance card at top of page.

I was trying to find a way for the monthly period to run from 26th to 26th, instead of 1st to 1st

thanks

**Comment by JC5** [comment]
https://docs.firefly-iii.org/firefly-iii/about-firefly-iii/what-its-not/#changing-the-monthly-cycle-to-another-day-of-the-month

**Reply by chilledhobbit** [reply]
Is this something you will add support for in the future?
